src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenDynamicTests.cs

```diff
@@ -1489,6 +1491,119 @@ .maxstack  13
         #region Conversions
 
         [Fact]
+        public void LocalFunctionArgumentConversion()
+        {
+            string src = @"
+using System;
+public class C
+{
+    static void Main()
+    {
+        int capture1 = 0;
+        void L1(int x) => Console.Write(x);
+        Action<int> L2(int x)
+        {
+            Console.Write(capture1);
+            Console.Write(x);
+            void L3(int y)
+            {
+                Console.Write(y + x);
+            }
+            return L3;
+        }
+        dynamic d = 2;
+        L1(d);
+        dynamic l3 = L2(d);
+        l3(d);
+    }
+}";
+            CompileAndVerifyExperimental(src,
+                expectedOutput: "2024",
+                additionalRefs: new[] { SystemCoreRef, CSharpRef }).VerifyDiagnostics();
+            CompileAndVerifyIL(src, "C.Main",
+                parseOptions: TestOptions.ExperimentalParseOptions,
```

Comment:
Please use options specific to local functions so that it would be straightforward to adjust the tests once we remove the feature flag for local functions.